tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -84,31 +84,20 @@ function configure_different_shims_base() {
 
 	mkdir -p /usr/local/bin
 
-	for shim in "${shims[@]}"; do
-		local shim_binary="containerd-shim-kata-${shim}-v2"
-		local shim_file="/usr/local/bin/${shim_binary}"
-		local shim_backup="/usr/local/bin/${shim_binary}.bak"
-
-		if [ -f "${shim_file}" ]; then
-			echo "warning: ${shim_binary} already exists" >&2
-			if [ ! -f "${shim_backup}" ]; then
-				mv "${shim_file}" "${shim_backup}"
-			else
-				rm "${shim_file}"
-			fi
-		fi
-
-		cat << EOF | tee "$shim_file"
-#!/usr/bin/env bash
-KATA_CONF_FILE=/opt/kata/share/defaults/kata-containers/configuration-${shim}.toml /opt/kata/bin/containerd-shim-kata-v2 "\$@"
-EOF
-		chmod +x "$shim_file"
-
-		if [ "${shim}" == "${default_shim}" ]; then
-			echo "Creating the default shim-v2 binary"
-			ln -sf "${shim_file}" /usr/local/bin/containerd-shim-kata-v2
+	local shim_binary="containerd-shim-kata-v2"
+	local shim_file="/usr/local/bin/${shim_binary}"
+	local shim_backup="/usr/local/bin/${shim_binary}.bak"
+
+	if [ -f "${shim_file}" ]; then
+		echo "warning: ${shim_binary} already exists" >&2
+		if [ ! -f "${shim_backup}" ]; then
```

Comment:
Can this be simplified by `mv "${shim_file}" "${shim_backup}"`in spite of whether the backup file exists or not?

Reply:
And `cleanup_different_shims_base` should be updated too.